tensorflow/lite/micro/recording_micro_allocator.cc

```diff
@@ -176,8 +178,12 @@ TfLiteStatus RecordingMicroAllocator::AllocateNodeAndRegistrations(
     // potential for fragmentation, manually adjust the accounting by
     // decrementing by 1 and adding the actual number of nodes used in the
     // graph:
-    recorded_node_and_registration_array_data_.count +=
-        model->subgraphs()->Get(subgraph_idx)->operators()->size() - 1;
+    if (model->subgraphs()->Get(subgraph_idx)->operators()) {
+      recorded_node_and_registration_array_data_.count +=
+          model->subgraphs()->Get(subgraph_idx)->operators()->size() - 1;
+    } else {
+      recorded_node_and_registration_array_data_.count -= 1;
+    }
```

Comment:
It is possible for a subgraph to have no operators. An example is an `IF` node where only the true-branch has a body and the else-branch is empty. In that case `operators()` returns a nullpointer and this would segfault.